src/transport_none.h

```diff
@@ -55,6 +55,11 @@ typedef int shm_internal_datatype_t;
 typedef int shm_internal_op_t;
 typedef int shmem_transport_ct_t;
 
+struct shmem_transport_ctx_t{};
```

Comment:
Woah, does this work?  First time I've seen it.

Reply:
I'm wondering if we should just declare it to be ```void *``` since it's never dereferenced in transport-none (maybe same in the Portals transport)?

Reply:
Haha, yeah.  I was kinda surprised too, but it seems to work.  But yes, I'll just do `void *` since it simpler/probably more portable.